pkg/pillar/zedcloud/send.go

```diff
@@ -786,15 +786,9 @@ func SendOnIntf(workContext context.Context, ctx *ZedCloudContext, destURL strin
 			log.Tracef("SendOnIntf to %s, response %s\n", reqUrl, resp.Status)
 			return resp, contents, senderStatus, nil
 		default:
-			errStr := fmt.Sprintf("SendOnIntf to %s reqlen %d statuscode %d %s",
+			errStr := fmt.Sprintf("SendOnIntf to %s reqlen %d statuscode %d %s body: %s",
 				reqUrl, reqlen, resp.StatusCode,
-				http.StatusText(resp.StatusCode))
-			// zedrouter probing sends 'http' to zedcloud server, expect to get status of 404, not an error
-			if resp.StatusCode != http.StatusNotFound || ctx.AgentName != "zedrouter" {
-				log.Errorln(errStr)
-				log.Errorf("Got payload for status %s: %s",
```

Comment:
We should keep here `log.Error` call for `errStr` (unless it is logged later from somewhere else but I wouldn't rely on it). We want to have this error included both in the logs and in the console output.

Reply:
I left it in there now.